sympy/integrals/laplace.py

```diff
@@ -382,7 +382,7 @@ def _laplace_build_rules():
     (sin(2*sqrt(a*t))/t, pi*erf(sqrt(a/s)),
      S.true, S.Zero, dco), # 4.7.34
     (cos(omega*t), s/(s**2+omega**2),
-     S.true, Abs(im(omega)), dco), # 4.7.43
+      S.true, Abs(im(omega)), dco), # 4.7.43
```

Comment:
```suggestion
     S.true, Abs(im(omega)), dco), # 4.7.43
```
I think that these should better stay same or it will appear in the diff